src/Pages/Account/Login/Components/ButtonLogin/ButtonLogin.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+class ButtonLogin extends Component {
+  render() {
+    return (
+      <>
+        <Button__login>
+          <LoginBtn__text>Log In</LoginBtn__text>
+        </Button__login>
+      </>
+    );
+  }
+}
+
+const Button__login = styled.button`
```

Comment:
```suggestion
const Button = styled.button`
```
복잡한 컴포넌트가 아니라면 태그 이름 첫글자 대문자로 써주는 것도 가능합니다

Reply:
넵, 변경했습니다